src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -951,8 +951,6 @@ static int client_timeout_resend(sd_event_source *s, uint64_t usec, void *userda
         assert(s);
         assert(client->event);
 
-        (void) event_source_disable(client->timeout_resend);
```

Comment:
This function can return before re-enabling this, won't it be a problem?

Reply:
When it returns earlier, then it calls `client_set_state()` or `client_stop()`, and both disable the timer. Moreover, timer event source is one shot by default, so it is not necessary to disable it in the callback.

Reply:
The commit message now mention about that.